### ISSUE TITLE
Bug: encrypting/decrypting large integers results in mismatched digits

### ISSUE THREAD

**Issue by psvet** [TechnologyAdvice/Cryptex#5]
Found when trying to store a Salesforce client secret.

Here's an example using a 19-digit integer (the length of the client secret):
![screen shot 2016-07-11 at 2 32 42 pm](https://cloud.githubusercontent.com/assets/7945073/16747575/78c58d18-4774-11e6-852c-32990ef20d12.png)

Notice the `400` ending the decrypted string, versus the `500` of the original. The same thing occurs using `getsecret` after storing the string in the config file.

Compare that with appending a letter to the integer, which works as expected:
![screen shot 2016-07-11 at 2 39 54 pm](https://cloud.githubusercontent.com/assets/7945073/16747792/6d33dcec-4775-11e6-9c51-79be589c12d7.png)

**Comment by TomFrost** [comment]
Replicated using the plaintext algorithm... this is wild:

```
╭─<tom@tom-ta>-<~/Projects/Cryptex>-<1bf19ad∙master∙▪>-<9:43PM>-◇
╰─➤ node src/cli.js encrypt 1234567890123400500
1234567890123400400
```

**Comment by TomFrost** [comment]
Looks like the issue is Javascript, rounding longs big-endian style.

```
> const int = 123456789012340045
undefined
> int
123456789012340050
```

**Comment by TomFrost** [comment]
The underlying issue is yargs parsing too-long numbers as a number instead of a string. Bug report logged here: https://github.com/yargs/yargs/issues/550

In absence of any near-term motion on that ticket, I'll write a workaround that manually parses encrypt arguments from the original process.argv.

**Comment by TomFrost** [comment]
Fixed in v1.0.0. Thanks, yargs team!
